### ISSUE TITLE
add wiki chapters for pkg filters and orderers

### ISSUE THREAD

**Issue by nerdvegas** [nerdvegas/rez#1217]

**Comment by JeanChristopheMorinPerso** [comment]
Note to whoever will work on this: `package_orderers` are partially documented in https://github.com/nerdvegas/rez/wiki/Configuring-Rez#package_orderers.

`package_filters` are technically documented in https://github.com/nerdvegas/rez/wiki/Configuring-Rez#package_filter.

**Comment by nerdvegas** [comment]
Yeah I should point out, this is true for lots of stuff actually. There
will be a small outline of the config setting's functionality (and this is
parsed out of the rezconfig.py sourcefile); if the issue is
important/complex enough, it should be expanded on further in a dedicated
wiki section somewhere.

Btw this is why this comment is here:
https://github.com/nerdvegas/rez/blob/master/src/rez/rezconfig.py#L43-L44
